src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/CustomRealtimeBlurView.Android.cs

```diff
@@ -0,0 +1,428 @@
+		private Paint mPaint;
+		private readonly Rect mRectSrc = new Rect();
+		private readonly Rect mRectDst = new Rect();
+		// mDecorView should be the root view of the activity (even if you are on a different window like a dialog)
+		private View mDecorView;
+		// If the view is on different root view (usually means we are on a PopupWindow),
+		// we need to manually call invalidate() in onPreDraw(), otherwise we will not be able to see the changes
+		private bool mDifferentRoot;
+		private static int RENDERING_COUNT;
+		private static int BLUR_IMPL;
+
+		private readonly PreDrawListener _preDrawListener;
+
+		public RealtimeBlurView(Context context) : base(context)
+		{
+			_preDrawListener = new PreDrawListener(this);
+			mBlurImpl = GetBlurImpl(); // provide your own by override getBlurImpl()			
+			mBlurRadius = TypedValue.ApplyDimension(ComplexUnitType.Dip, 10, context.Resources.DisplayMetrics);
+			//mDownsampleFactor = a.getFloat(R.styleable.RealtimeBlurView_realtimeDownsampleFactor, 4);
+
+			mPaint = new Paint();
+		}
+
+		public void SetOverlayColor(int color, bool invalidate = true)
+		{
+			if (mOverlayColor != color)
+			{
+				mOverlayColor = color;
+				if (invalidate)
+				{
+					Invalidate();
+				}
+			}
+		}
+
+		protected IBlurImpl GetBlurImpl()
+		{
+			if (BLUR_IMPL == 0)
+			{
+				// try to use stock impl first
+				if (Build.VERSION.SdkInt >= BuildVersionCodes.JellyBeanMr1)
+				{
+					try
+					{
+						AndroidStockBlurImpl impl = new AndroidStockBlurImpl();
+						Bitmap bmp = Bitmap.CreateBitmap(4, 4, Bitmap.Config.Argb8888);
+						impl.Prepare(Context, bmp, 4);
+						impl.Release();
+						bmp.Recycle();
+						BLUR_IMPL = 3;
+					}
+					catch (Exception e)
+					{
+						if (this.Log().IsEnabled(Microsoft.Extensions.Logging.LogLevel.Error))
+						{
+							this.Log().Error("Failed GetBlurImpl", e);
+						}
+					}
+				}
+			}
+
+			if (BLUR_IMPL == 0)
+			{
+				// fallback to empty impl, which doesn't have blur effect
+				BLUR_IMPL = -1;
+			}
+			switch (BLUR_IMPL)
+			{
+				case 3:
+					return new AndroidStockBlurImpl();
+				default:
+					return new EmptyBlurImpl();
+			}
+		}
+
+		private class PreDrawListener : Java.Lang.Object, ViewTreeObserver.IOnPreDrawListener
+		{
+			private readonly JniWeakReference<RealtimeBlurView> _weakBlurView;
+
+			public PreDrawListener(RealtimeBlurView blurView)
+			{
+				_weakBlurView = new JniWeakReference<RealtimeBlurView>(blurView);
+			}
+
+			public PreDrawListener(IntPtr handle, JniHandleOwnership transfer)
+				: base(handle, transfer)
+			{
+			}
+
+			public bool OnPreDraw()
+			{
+				if (!_weakBlurView.TryGetTarget(out var blurView))
+				{
+					return false;
+				}
+
+				int[] locations = new int[2];
+				Bitmap oldBmp = blurView.mBlurredBitmap;
+				View decor = blurView.mDecorView;
+				if (decor != null && blurView.IsShown && blurView.Prepare())
+				{
+					bool redrawBitmap = blurView.mBlurredBitmap != oldBmp;
+					oldBmp = null;
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'oldBmp'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)